### ISSUE TITLE
[Select] Low Contrast Indicator in Select Navigation (Keyboard Interaction)

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#2705]
### Provide a general summary of the issue here

**Description:**
When navigating between page numbers in the select using keyboard arrow keys, the grey rectangle that indicates the current focus position has a contrast ratio of only 1.1:1, making it very difficult to see. This fails accessibility standards and should be improved.
The same issue occurs in dark mode, where the contrast ratio is 1.6:1.

**Link to finding:** https://github.com/sl-design-system/components/issues/2514#issuecomment-3319119564

**Expected Behavior:**
The focus indicator should have a minimum contrast ratio of 3:1 to meet accessibility guidelines.

**Inspiration:**

<img width="349" height="490" alt="Image" src="https://github.com/user-attachments/assets/c21c7707-5cbd-4aaf-959c-4fdf76d0f1d8" />

**Tasks**
- [x] improve design in figma
- [x] update hover state in figma component
- [ ] implement in code

### 👤 Your name

DR

### 🧢 Your Product/Team

SLDS

**Comment by RoaldBoerema** [comment]
Be aware there is a 1px border on the left side of the indicator to make it a little bit darker

**Comment by Diaan** [comment]
This is already fixed, right?

**Comment by a11ymiko** [comment]
In Select version 2.1.4 this issue is fixed. 
Focus indicator looks like:

<img width="1206" height="333" alt="Image" src="https://github.com/user-attachments/assets/e09c6c4b-6dfb-4541-b4fe-d37d19e5d534" />
